playground/components/framework-switcher.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import { Frameworks } from '@/types';
+import { useSandpack } from '@codesandbox/sandpack-react';
+import { Dispatch, SetStateAction } from 'react';
```

Comment:
💡 idea: 
```
import type { Frameworks } from '@/types';
import { useSandpack } from '@codesandbox/sandpack-react';
import type { Dispatch, SetStateAction } from 'react';
```

This tells the compiler these imports are not needed during runtime.